projects/demo-integrations/cypress/tests/kit/date/date-min-max.cy.ts

```diff
@@ -0,0 +1,103 @@
+import {DemoPath} from '@demo/routes';
+
+describe('Date', () => {
+    describe('Max date', () => {
+        beforeEach(() => {
+            cy.visit(`/${DemoPath.Date}/API?max=2020-05-05`);
+            cy.get('#demo-content input')
+                .should('be.visible')
+                .first()
+                .focus()
+                .as('input');
+        });
+
+        it('Input less than max value', () => {
+            cy.get('@input')
+                .type('31122019')
+                .should('have.value', '31.12.2019')
+                .should('have.prop', 'selectionStart', '31.12.2019'.length)
+                .should('have.prop', 'selectionEnd', '31.12.2019'.length);
+        });
+
+        it('05.12.202| => type 5 => 05.05.2020 (max value)', () => {
+            cy.get('@input')
+                .type('0512202')
+                .should('have.value', '05.12.202')
+                .should('have.prop', 'selectionStart', '05.12.202'.length)
+                .should('have.prop', 'selectionEnd', '05.12.202'.length)
+                .type('5')
+                .should('have.value', '05.05.2020')
+                .should('have.prop', 'selectionStart', '05.05.2020'.length)
+                .should('have.prop', 'selectionEnd', '05.05.2020'.length);
+        });
+
+        it('0|3.06.2020 => type 7 => 05|.05.2020 (max value)', () => {
+            cy.get('@input')
+                .type('03062020')
+                .type('{leftArrow}'.repeat('3.06.2020'.length))
```

Comment:
```suggestion
                .type('{moveToStart}{rightArrow}')
```

It does the same but it more readable (at least I hope so)